features/FxA-46-sms-install-link/README.md

```diff
@@ -0,0 +1,54 @@
+# SMS Install Link
+
+https://mozilla.aha.io/features/FXA-46
+
+## Stories
+
+As a user signing up for a Firefox Account, I may want to easily install
+Firefox on my Android or iOS device, removing the friction of searching
+by sending me the link in a text message.
+
+## Success Criteria
+
+This feature will be successful if we measurable increase the rate of
+FIrefox activations for Android and iOS.
```

Comment:
We should definitely watch this, but since we don't control the gathering and reporting of it, I'm hesitant to make "increase in mobile firefox installs" our primary metric.

We could substitute a more accounts-specific one along the lines of "increase percentage of FxA users that have multiple devices connected", which I think is closer to the end goal of what we're trying to drive here.